### PR TITLE
Update short description and logo

### PR DESCRIPTION
**Pull request by Julievrz**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Update short description and logo
| Type?         | improvement
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | https://github.com/PrestaShop/PrestaShop/issues/15858
| How to test?  | Please check that short description and logo are updated.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## About
 
-Adds a list of the best customers to the Stats dashboard.
+Enrich the ‘Stats’ section of your back office, add a list of the best customers to the dashboard.
```
Comment (PierreRambaud) [inline Readme.md:5]: Is there a reason to use `‘` this char?
```suggestion
Enrich the "Stats" section of your back office, add a list of the best customers to the dashboard.
```

Comment (Julievrz) [inline Readme.md:5]:
```suggestion
Enrich your stats, add a list of the best customers to the dashboard.
```

File: statsbestcustomers.php [hunk 2]
```diff
@@ -95,7 +95,7 @@ public function __construct()
         );
 
         $this->displayName = $this->trans('Best customers', array(), 'Modules.Statsbestcustomers.Admin');
-        $this->description = $this->trans('Adds a list of the best customers to the Stats dashboard.', array(), 'Modules.Statsbestcustomers.Admin');
+        $this->description = $this->trans('Enrich the ‘Stats’ section of your back office, add a list of the best customers to the dashboard.', array(), 'Modules.Statsbestcustomers.Admin');
```
Comment (Julievrz) [inline statsbestcustomers.php:98]:
```suggestion
        $this->description = $this->trans('Enrich your stats, add a list of the best customers to the dashboard.', array(), 'Modules.Statsbestcustomers.Admin');
```

**Review by PierreRambaud — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thank @Julievrz

**Pull request merged**